src/state/StorageContext.js

```diff
@@ -79,7 +79,8 @@ const defaultDapps = [
 export default function StorageContextProvider({ children }) {
   const { mySky, user } = React.useContext(AuthContext);
   const [state, setState] = React.useState(initialState);
-  const { dapps, isStorageProcessing } = state;
+  const { dapps, isStorageProcessing, isStorageInitialised } = state;
+  const preventParallelStorageAccess = isStorageProcessing || isStorageInitialised;
```

Comment:
won't this make it so `preventParallelStorageAccess` is `true` after the storage has long been initialized?

Reply:
```suggestion
  const preventParallelStorageAccess = isStorageProcessing || !isStorageInitialised;
```